packages/create-react-native-library/templates/common/$.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+on:
+  push:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup
+        uses: ./.github/actions/setup
+
+      - name: Lint files
+        run: yarn lint
+
+      - name: Typecheck files
+        run: yarn typescript
```

Comment:
Putting lint and typecheck in one job is intended? It used to be separate. However, libraries are small, so I'd expect this to be actually faster than spinning up the whole env twice to run them separately

Reply:
@thymikee yeah just to reduce setting up another job since it needs to clone repo and install deps all over again unlike circle ci